language/italian/form_validation_lang.php

```diff
@@ -15,7 +16,8 @@
 $lang['form_validation_valid_email'] = 'Il campo {field} deve contenere un indirizzo email valido.';
 $lang['form_validation_valid_emails'] = 'Il campo {field} deve contenere tutti indirizzi email validi.';
 $lang['form_validation_valid_url'] = 'Il campo {field} deve contenere un URL valido.';
-$lang['form_validation_valid_ip'] = 'Il campo {field} deve contenenre un indirizzo IP valido.';
+$lang['form_validation_valid_ip'] = 'Il campo {field} deve contenere un indirizzo IP valido.';
+$lang['form_validation_valid_base64'] = 'Il campo {field} deve contenere una stringa in Base64.';
```

Comment:
@jim-parry please merge this 2 lines.  

form_validation_valid_ip:  fix translation
form_validation_valid_base64: missing translation